### ISSUE TITLE
When I train the model, I found the training loss is not stable

### ISSUE THREAD

**Issue by vandesa003** [backseason/PoolNet#37]
Hi @backseason @Andrew-Qibin , thanks for your great work! I am trying to retrain the model, but the training loss going with a large fluctuation. Is that because of no BN layer or need a pre-trained backbone? By the way, could you please provide the pre-trained ResNet or VGG model since you modified the network. looking forward to your reply.

**Comment by backseason** [comment]
It's normal as we do not use the mean of the loss and the spatial sizes of input images vary.

**Comment by vandesa003** [comment]
Thanks for your reply! have you done some experiment about the fusion of feature maps? I've noticed that some works used channel-wise concatenation while your work used pixel-wise add. Is pixel-wise add fusion always outperform channel-wise concatenation?

**Comment by backseason** [comment]
Oh my mistake, the pre-trained models for ResNet-50 and VGG-16 are already provided here: https://drive.google.com/open?id=1Q2Fg2KZV8AzNdWNjNgcavffKJBChdBgy.

We have tried concatenation but found nearly the same as pixel-wise add.
So for simpleness we use pixel-wise add.

**Comment by vandesa003** [comment]
@backseason Noted with thanks! I'm going to close this issue then.
